src/main/kotlin/no/elhub/devxp/autochangelog/AutoChangelog.kt

```diff
@@ -0,0 +1,119 @@
+package no.elhub.devxp.autochangelog
```

Comment:
:100: For being a "full rewrite" this is an incredibly tidy changeset. Nice work!